src/main/webapp/script.js

```diff
@@ -303,3 +308,22 @@ async function getEvents(url, index, option) {
 function openEvent() {
   window.location.pathname = '/display-event.html'
 }
+
+/**
+ * Toggles the display of the survey page to indicate which option is selected,
+ * while saving the survey responses.
+ *
+ * @param {number} question The question number to apply the toggle.
+ * @param {number} index Which bubble on the question to apply the toggle to.
+ */
+var surveyResponses = [-1, -1, -1, -1, -1];
+function toggleSurveyDisplay(question, index) {
+	const circle = document.getElementsByClassName('survey-select')[question*5 + index];
+	if(surveyResponses[question] >= 0) {
+		var oldIndex = surveyResponses[question];
+		const oldCircle = document.getElementsByClassName("survey-select")[question*5 + oldIndex];
+		oldCircle.style.backgroundColor = "transparent";
+	}
+	circle.style.backgroundColor = "white";
```

Comment:
In JS I think we're supposed to use single quotes instead of double

Reply:
Maybe useful to use a linter to automatically check the code against the style guide:

https://github.com/eslint/eslint
https://github.com/google/eslint-config-google